packages/interactor/src/create-interactor.ts

```diff
@@ -18,8 +19,11 @@ export function createInteractor<E extends Element>(interactorName: string) {
           if(args.length) {
             actionDescription += ` with ` + args.map((a) => JSON.stringify(a)).join(', ');
           }
-          return interaction(`${actionDescription} on ${this.description}`, () => {
-            return converge(() => {
+          return interaction(`${actionDescription} on ${this.description}`, async () => {
+            if(bigtestGlobals.runnerState === 'assertion') {
+              throw new Error(`tried to ${actionDescription} on ${this.description} in an assertion, actions should only be performed in steps`);
+            }
```

Comment:
Another option would have been to implement this in `Interactor#perform`, but I liked this option better.